docs/reference-implementation/Getting-Started.md

```diff
@@ -181,7 +181,7 @@ xxx-xxxxxxx-xxxxxxx-xxxx
 # Verify that this is indeed the subscription you want to target. Otherwise you can switch the scope using:
 # az account set --subscription <name>
 
-# Make sure to change the name to a unique one within your tenant
+# Make sure to change the name to a unique one within your tenant. You must have Azure CLI version 2.25.0 or later.
```

Comment:
How about adding this requirement also into the pre-requisites section? As a bracket to the link to Azure CLI maybe?